repos/fdbt-site/src/design/modules/_popups.scss

```diff
@@ -0,0 +1,23 @@
+.popup {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100vh;
+    background-color: rgba(0, 0, 0, 0.2);
```

Comment:
Could this be a govuk color? or are there no appropriate govuk ones?

Reply:
Good catch, typically in regular CSS, the `rgba` doesn't accept hex colours, but in Sass, hex colours work fine! I've updated it.